slick/src/main/scala/slick/jdbc/JdbcActionComponent.scala

```diff
@@ -208,17 +203,23 @@ trait JdbcActionComponent extends SqlActionComponent { self: JdbcProfile =>
         case ParameterSwitch(cases, default) =>
           findSql(cases.find { case (f, n) => f(param) }.map(_._2).getOrElse(default))
       }
-      (tree match {
+      val (comment, tree2) = tree match {
+        case Comment(comment, child) =>
+          (s"/* $comment */\n", child)
```

Comment:
Do we really want to be able to add empty comments?

Reply:
Added check for empty comment